### ISSUE TITLE
wallet logout doesnt logout the user

### ISSUE THREAD

**Issue by ashachaf** [bancorprotocol/webapp#693]
#### flow:
open bancor.network
click to connect wallet
see this:
![image](https://user-images.githubusercontent.com/40430285/102011989-76846c00-3d50-11eb-983d-0655c2fb51b1.png)
click to logout

#### results:
actual: nothing happens
expected: logout and disconnect my wallet from the app

#### fix:
onboard.js support this options with this code:

You may want to reset all of Onboard's internal wallet state and also disconnect from any active SDK instances when a user logs out of your app. You can call the walletReset function to do this easily.
```
// user wants to log out of session and the wallet state needs to be reset...
onboard.walletReset()
// this method is synchronous and returns undefined
```

**Comment by pingustar** [comment]
done.

**Comment by Velua** [comment]
@pingustar this might have returned, any ideas?

**Comment by pingustar** [comment]
fixed in 42e85d2949ec3d815e354f8093c5fb5241afbaea
